admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/config/MosipEnvironment.java

```diff
@@ -0,0 +1,25 @@
+package io.mosip.kernel.syncdata.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.EnvironmentAware;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+
+@Configuration
+public class MosipEnvironment implements EnvironmentAware {
+
+	@Autowired
+	private Environment environment;
```

Comment:
directly inject required property with @Value("${}") in the required class, Please remove this class